spec/rubocop/cop/rails/require_dependency_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::RequireDependency, :config do
+  context 'when require' do
+    let(:source) do
+      <<~RUBY
+        require 'foo'
+      RUBY
+    end
+
+    it { expect_no_offenses source }
```

Comment:
Can you tweak the test code style?

```suggestion
    it 'does not register an offense' do
      expect_no_offenses(<<~RUBY)
        require 'foo'
      RUBY
    end
```

Reply:
Might be good for this repo to have a cop that enforces this style 😄

Reply:
Yeah, I thought the same thing :-)